Library/Homebrew/keg_relocate.rb

```diff
@@ -67,9 +67,13 @@ def text_files
     path.find do |pn|
       next if pn.symlink? || pn.directory?
       next if Metafiles::EXTENSIONS.include? pn.extname
-      if Utils.popen_read("/usr/bin/file", "--brief", pn).include?("text") ||
-         pn.text_executable?
-        text_files << pn
+      # File has known issues with reading files on other locales.
+      # http://bugs.gw.com/view.php?id=292
+      with_custom_locale("C") do
```

Comment:
It's a minor concern, but I'm wondering how much switching back and forth environments for every single file affects performance. I think we could safely move this outside of `path.find do` without adverse effects. The only disadvantage would be that it's a bit more disconnected from the `/usr/bin/file` invocation.